recipes-app/node-red-gpio/node-red-gpio_0.2.1-IOT2050.bb

```diff
@@ -14,15 +14,9 @@ inherit dpkg-raw
 DESCRIPTION = "node-red-gpio-integration"
 MAINTAINER = "chao.zeng@siemens.com"
 
-SRC_URI = " \
-    git://github.com/node-red/node-red-nodes;protocol=https \
-    file://0001-add-the-board-info-add-the-led-control-node.patch \
-    file://0002-extend-gpio-to-D19.patch \
-    file://0003-Clean-up-mraa-objects-on-node-closing.patch \
-    file://0004-Add-USER-button.patch \
-    file://0005-Add-configurable-GPIO-mode-for-DIN.patch \
-    file://0006-node-red-gpio-Add-support-for-initial-message-from.patch"
-SRCREV="3087e8e2a1ea189f394bca0a2af159ad859d7722"
+SRC_URI = "git://github.com/node-red/node-red-nodes;protocol=https"
+
+SRCREV="e4dc3a2bcfd38dfd18eff1bfa5542ff56c40f27c"
```

Comment:
Also here: recipe version is now no longer "-IOT2050" but actually some upstream version + git.